src/components/Button/Button.spec.tsx

```diff
@@ -30,6 +30,12 @@ describe('Button', () => {
     expect(await driver.isMobile()).toEqual(true);
   });
 
+  it('should let focus on inner button using ref', async () => {
+    const ref = React.createRef<HTMLButtonElement>();
+    const driver = createDriver(<Button fullWidth ref={ref} />);
+    expect(ref.current.focus).toBeInstanceOf(Function);
```

Comment:
instead of checking the `focus` function, lets see that the `ref.current` is an instance of the button-next

Reply:
> @eranshabi instead of checking the `focus` function, lets see that the `ref.current` is an instance of the button-next

good comment, I changed the test and found out that the `focus` method that I get is actually of `focusableHOC`.
I did PR to fix this in wix-ui-core:
https://github.com/wix/wix-ui/pull/1321